src/renderer/components/chart/bar-chart.tsx

```diff
@@ -116,9 +116,7 @@ export class BarChart extends React.Component<Props> {
         position: "cursor",
         callbacks: {
           title: tooltipItems => {
-            const now = new Date().getTime();
-
-            if (new Date(tooltipItems[0].xLabel).getTime() > now) return "";
+            if (new Date(tooltipItems[0].xLabel).getTime() > Date.now()) return "";
 
             return `${tooltipItems[0].xLabel}`;
           },
```

Comment:
`xLabel` can be undefined, so I think that we should return `""` if it is undefined.

```typescript
title(([{xLabel}])) {
  const labelDate = new Date(xLabel);

  return xLabel && labelDate.getTime() <= Date.now()
    ? xLabel.toString()
    : "";
}
```

Reply:
Thanks for spotting the issue! Fixed.